Documentation/joss/paper.md

```diff
@@ -1,15 +1,5 @@
-
-Note to OIP developers: observe the [guidelines](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain) and our [notes](./notes.md).
-
-Most important instructions:
-
-> 250-1000 words
-> explain the software functionality and domain of use to a *non-specialist* reader
-
-See [example](http://bit.ly/2x22gxT) and [a more complex example](https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography).
-
 ---
-title: 'TUM OpenInfraPlatform: open source package for simultaneous viewing and analysis of Industry Foundation Classes (IFC) and Point Cloud Data (PCD) models'
+title: 'TUM Open Infra Platform: open source package for simultaneous viewing and analysis of digital models in the civil engineering domain'
```

Comment:
what are digital modesl? -> "building information models"

Reply:
Well, IMHO, building information models kind of disclose point clouds ... and we can open OFF files (since recently), which is pure geometry ... I'm in search of a common denominator here. Suggestions welcome.